### ISSUE TITLE
Should Syntax does not remove extra parenthesis

### ISSUE THREAD

**Issue by yvele** [skovhus/jest-codemods#48]
This line:

```js
foo.should.be.true()
```

is transformed into:

```js
expect(foo).toBe(true)();
```

but should be

```js
expect(foo).toBe(true);
```

PS: As a dirty workaround I just replace `)();` with `);`

**Comment by skovhus** [comment]
Thanks for reporting this!

**Comment by skovhus** [comment]
Fix released as `jest-codemods@0.9.2`
